cli/src/torii/routing.rs

```diff
@@ -387,13 +387,10 @@ async fn handle_metrics(sumeragi: Arc<Sumeragi>, network: Addr<IrohaNetwork>) ->
 
 #[cfg(feature = "telemetry")]
 async fn handle_status(sumeragi: Arc<Sumeragi>, network: Addr<IrohaNetwork>) -> Result<Json> {
-    /*
-    update_metrics(&wsv, network).await?;
-    let status = Status::from(&wsv.metrics);
+    sumeragi.update_metrics(network);
+    let status = Status::from(&sumeragi.get_clone_of_world_state_view().metrics);
```

Comment:
Naming convention

```suggestion
    let status = Status::from(&sumeragi.wsv_clone().metrics);
```

Or even 


```suggestion
    let status = Status::from(&sumeragi.wsv().clone().metrics);
```

And even better, following the example of our FFI function, 
```suggestion
let status = sumeragi.update_metrics(network).clone().into()
```
i.e.  `update_metrics` returns `&Metrics`, which you can clone.

Reply:
using suggestion 1